package-lock.json

```diff
@@ -14,18 +14,18 @@
       }
```

Comment:
package-lock is updated only before release. No point to update it for every PR.

Reply:
Sorry, I tried to add new dependencies while performing tasks